packages/flutter_tools/lib/src/flx.dart

```diff
@@ -50,6 +50,12 @@ class _Asset {
 
   final String source;
 
+  File get assetFile {
+    return new File(source != null ? '$base/$source' : '$base/$relativePath');
+  }
+
+  bool get assertFileExists => assetFile.existsSync();
```

Comment:
I think this should just be called fileExists if it's not actually asserting anything.